src/csharp/Grpc.Tools.nuspec

```diff
@@ -2,32 +2,70 @@
 <package>
   <metadata>
     <id>Grpc.Tools</id>
-    <title>gRPC C# Tools</title>
-    <summary>Tools for C# implementation of gRPC - an RPC library and framework</summary>
-    <description>Precompiled protobuf compiler and gRPC protobuf compiler plugin for generating gRPC client/server C# code. Binaries are available for Windows, Linux and MacOS.</description>
+    <title>gRPC Build Tools</title>
+    <summary>gRPC and Protocol Buffer compiler for managed C# and native C++ projects.</summary>
+    <description>
+      Add this package to a project that contains .proto files to be compiled to code.
+      It contains the compilers, include files and project system integration for gRPC
+      and Protocol buffer service description files necessary to build them on Windows,
+      Linux and MacOS. Managed runtime is supplied separately in the Grpc.Core package.
+    </description>
     <version>$version$</version>
     <authors>Google Inc.</authors>
     <owners>grpc-packages</owners>
     <licenseUrl>https://github.com/grpc/grpc/blob/master/LICENSE</licenseUrl>
     <projectUrl>https://github.com/grpc/grpc</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <releaseNotes>Release $version$</releaseNotes>
-    <copyright>Copyright 2015, Google Inc.</copyright>
+    <copyright>Copyright 2015 gRPC authors</copyright>
```

Comment:
@jtattermusch: Should it be "Google" or "gRPC authors"? It varies.

Reply:
"gRPC authors" is correct. We probably did't update all the occurrences.